[2025-02-26 12:03:59,179:INFO:main:welcome]
[2025-03-03 14:19:46,849:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 14:19:46,879:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 14:19:46,922:INFO:common:created directoty at : artifacts]
[2025-03-03 14:19:46,922:INFO:common:created directoty at : artifacts/data_ingestion]
[2025-03-03 14:21:22,726:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 14:21:22,731:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 14:21:22,735:INFO:common:created directoty at : artifacts]
[2025-03-03 14:21:22,739:INFO:common:created directoty at : artifacts/data_ingestion]
[2025-03-03 14:21:34,267:INFO:3529762362:artifacts/data_ingestion/data.zip downloaded]
[2025-03-03 14:42:56,998:INFO:main:>>>>>> stage Data_ingestion_stage started<<<<<<]
[2025-03-03 14:42:57,003:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 14:42:57,006:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 14:42:57,007:INFO:common:created directoty at : artifacts]
[2025-03-03 14:42:57,009:INFO:common:created directoty at : artifacts/data_ingestion]
[2025-03-03 14:43:08,207:INFO:data_ingestion:artifacts/data_ingestion/data.zip downloaded]
[2025-03-03 14:43:09,450:INFO:main:>>>>>> stage Data_ingestion_stage completed<<<<<<]
[2025-03-03 15:22:15,193:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:22:15,235:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:22:15,342:INFO:common:created directoty at : artifacts]
[2025-03-03 15:23:00,916:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:23:00,922:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:23:00,925:INFO:common:created directoty at : artifacts]
[2025-03-03 15:23:46,162:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:23:46,171:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:23:46,174:INFO:common:created directoty at : artifacts]
[2025-03-03 15:23:46,177:INFO:common:created directoty at : artifacts/prepare_base_model]
[2025-03-03 15:24:37,064:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:24:37,073:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:24:37,076:INFO:common:created directoty at : artifacts]
[2025-03-03 15:24:37,078:INFO:common:created directoty at : artifacts/prepare_base_model]
[2025-03-03 15:26:38,192:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:26:38,200:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:26:38,204:INFO:common:created directoty at : artifacts]
[2025-03-03 15:26:38,207:INFO:common:created directoty at : artifacts/prepare_base_model]
[2025-03-03 15:29:14,433:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:29:14,442:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:29:14,445:INFO:common:created directoty at : artifacts]
[2025-03-03 15:29:50,395:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:29:50,402:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:29:50,404:INFO:common:created directoty at : artifacts]
[2025-03-03 15:30:31,435:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:30:31,443:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:30:31,446:INFO:common:created directoty at : artifacts]
[2025-03-03 15:30:31,449:INFO:common:created directoty at : artifacts/prepare_base_model]
[2025-03-03 15:30:35,797:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-03-03 15:30:36,280:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-03-03 15:47:53,777:INFO:main:>>>>>> stage Data_ingestion_stage started<<<<<<]
[2025-03-03 15:47:53,794:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:47:53,800:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:47:53,802:INFO:common:created directoty at : artifacts]
[2025-03-03 15:47:53,803:INFO:common:created directoty at : artifacts/data_ingestion]
[2025-03-03 15:48:02,040:INFO:data_ingestion:artifacts/data_ingestion/data.zip downloaded]
[2025-03-03 15:48:02,819:INFO:main:>>>>>> stage Data_ingestion_stage completed<<<<<<]
[2025-03-03 15:48:02,819:INFO:main:*******************]
[2025-03-03 15:48:02,819:INFO:main:>>>>>> stage Prepare base model started <<<<<<]
[2025-03-03 15:48:02,819:ERROR:main:name 'PrepareBaseModelTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "E:\ML Folder\chicken_disease_classification\main.py", line 19, in <module>
    prepare_base_model = PrepareBaseModelTrainingPipeline()
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'PrepareBaseModelTrainingPipeline' is not defined
[2025-03-03 15:48:34,256:INFO:main:>>>>>> stage Data_ingestion_stage started<<<<<<]
[2025-03-03 15:48:34,262:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:48:34,270:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:48:34,272:INFO:common:created directoty at : artifacts]
[2025-03-03 15:48:34,273:INFO:common:created directoty at : artifacts/data_ingestion]
[2025-03-03 15:48:34,274:INFO:data_ingestion:artifacts/data_ingestion/data.zip already exists]
[2025-03-03 15:48:34,968:INFO:main:>>>>>> stage Data_ingestion_stage completed<<<<<<]
[2025-03-03 15:48:34,968:INFO:main:*******************]
[2025-03-03 15:48:34,968:INFO:main:>>>>>> stage Prepare base model started <<<<<<]
[2025-03-03 15:48:35,002:INFO:common:yaml_file:config\config.yaml loaded successfully]
[2025-03-03 15:48:35,010:INFO:common:yaml_file:params.yaml loaded successfully]
[2025-03-03 15:48:35,012:INFO:common:created directoty at : artifacts]
[2025-03-03 15:48:35,015:INFO:common:created directoty at : artifacts/prepare_base_model]
[2025-03-03 15:48:35,969:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-03-03 15:48:36,344:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-03-03 15:48:36,610:INFO:main:>>>>>> stage Prepare base model completed <<<<<<

x==========x]
